doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_cli.rst

```diff
@@ -21,6 +21,122 @@ Persistent storage
+
+* :kconfig:`CONFIG_BT_MESH_SHELL`
+* :kconfig:`CONFIG_BT_MESH_SHELL_LIGHT_CTRL_CLI`
+
+mdl_ctrl instance get-all
+	Print all instances of the Light LC Client model on the device.
+
+
+mdl_ctrl instance set <elem_idx>
+	Select the Light LC Client model instantiated on the specified element ID.
+	This instance will be used in message sending.
+	If no model instance is selected, the first model instance found on the device will be used by default.
+
+	* ``elem_idx`` - Element index where the model instance is found.
+
+
+mdl_ctrl mode-get
+	Get a Light Lightness Control Server's current Mode.
+
+
+mdl_ctrl mode-set <enable | disable>
+	Set a Light Lightness Control Server's current Mode and wait for a response.
+
+	* ``enable | disable`` - Mode to set.
+
+
+mdl_ctrl mode-set-unack <enable | disable>
+	Set a Light Lightness Control Server's current Mode without requesting a response.
+
+	* ``enable | disable`` - Mode to set.
+
+
+mdl_ctrl occupancy-get
+	Get a Light Lightness Control Server's current Occupancy Mode.
+
+
+mdl_ctrl occupancy-set <enable | disable>
+	Set a Light Lightness Control Server's current Occupancy Mode and wait for a response.
+
+	* ``enable | disable`` - Occupancy Mode to set.
+
+
+mdl_ctrl occupancy-set-unack <enable | disable>
+	Set a Light Lightness Control Server's current Occupancy Mode without requesting a response.
+
+	* ``enable | disable`` - Occupancy Mode to set.
+
+
+mdl_ctrl light-onoff-get
+	Get a Light Lightness Control Server's current OnOff state.
+
+
+mdl_ctrl light-onoff-set <onoff> [transition_time_ms [delay_ms]]
+	Tell a Light Lightness Control Server to turn the light on or off and wait for a response.
+
+	* ``onoff`` - OnOff state value to set. Use *on*, *enable*, or any non-zero value to enable the state.
+	* ``transition_time_ms`` - If present, defines the transition time in the message in milliseconds.
+	* ``delay_ms`` - If present, defines the delay in the message in milliseconds.
+
+
+mdl_ctrl light-onoff-set-unack <onoff> [transition_time_ms [delay_ms]]
+	Tell a Light Lightness Control Server to turn the light on or off without requesting a response.
+
+	* ``onoff`` - OnOff state value to set. Use *on*, *enable*, or any non-zero value to enable the state.
+	* ``transition_time_ms`` - If present, defines the transition time in the message in milliseconds.
+	* ``delay_ms`` - If present, defines the delay in the message in milliseconds.
+
+
+mdl_ctrl prop-get <id>
+	Get a Light Lightness Control Server property value.
+
+	* ``id`` - ID of the property to get. See :c:enum:`bt_mesh_light_ctrl_prop` for a list of supported properties.
+
+
+mdl_ctrl prop-set <id> <decimal> <frac>
+	Set a Light Lightness Control Server property value and wait for a response.
+
+	* ``id`` - ID of the property to set. See :c:enum:`bt_mesh_light_ctrl_prop` for a list of supported properties.
+	* ``decimal`` - Decimal part of the property value.
+	* ``frac`` - Fractional part of the property value.
```

Comment:
#7124 is merged now, so this should be updated to match.

Reply:
Done